frontend/simple/js/state.js

```diff
@@ -69,25 +94,45 @@ const mutations = {
     index === -1 && state.pending.push(contractId)
   }
 }
-
 // https://vuex.vuejs.org/en/getters.html
 // https://vuex.vuejs.org/en/modules.html
 const getters = {
   currentGroup (state) {
     return state[state.currentGroupId]
+  },
+  mailbox (state) {
+    return getMailbox(state)
+  },
+  unread (state) {
+    let mailbox = getMailbox(state)
+    return mailbox && state[ mailbox ].messages.filter((msg) => !msg.read).length
```

Comment:
Here too, the function name can be significantly improved to reduce confusion.

- `unread` does not tell me what the function does, or what it is returning. `unreadCount` would be better
- Nor does the name tell me tell me who this is for. unread ... of who / what? `mailboxUnreadCount` + take in the mailbox contract id as a param, would be way better